test/testpy/__init__.py

```diff
@@ -53,11 +53,15 @@ def AfterRun(self, result):
     # delete the whole tmp dir
     try:
       rmtree(self.tmpdir)
+      if "NODE_PIPE_DIR" in os.environ:
+        rmtree(os.environ["NODE_PIPE_DIR"]);
```

Comment:
Another wrinkle is that mktemp as an executable that can be run from the shell does not exist. 
For windows it looked to me like it already used a different directory for pips